### ISSUE TITLE
Issues using expected conditions over mobile locators (Selenium 4)

### ISSUE THREAD

**Issue by Juanap12** [appium/python-client#698]
## The problem

I'm trying to wait for an element to be clickable. This exact same wait does work for our web automation tests, but is showing the following error for mobile elements:

`AttributeError: type object 'Command' has no attribute 'IS_ELEMENT_DISPLAYED'`

This is the wait we're trying to perform

`element = WebDriverWait(parent, timeout, poll_frequency=poll,
                                 ignored_exceptions=[ElementNotVisibleException,
                                                     ElementNotSelectableException,
                                                     StaleElementReferenceException])\
                .until(expected_conditions.element_to_be_clickable(selector))`

"parent" is the driver
"timeout" is a value given in seconds
"selector" is a tuple that has the location strategy and the locator itself. e.g. `('-android uiautomator', 'new UiSelector().textContains("get started")')`

## Environment
- Appium version (or git revision) that exhibits the issue: Currently testing over `selenium>=4.1.3` and `Appium-Python-Client>=2.1.4`
- Desktop OS/version used to run Appium: macOS Monterey
- Node.js version (unless using Appium.app|exe):
- Mobile platform/version under test: Android 10
- Real device or emulator/simulator: emulator
- Appium CLI or Appium.app|exe: 1.22.0

## Details

Worth to mention that if I perform a `driver.find_element(*selector)` it does find the element. It shows the following result `<appium.webdriver.webelement.WebElement (session="a7f5a8a1-4e63-4847-bcc7-265666d7760b", element="00000000-0000-010c-ffff-ffff0000001e")>`. But, if I try to use the expected condition ".element_to_be_clickable" it shows the error mentioned at the beginning

**Comment by Juanap12** [comment]
I've tested using the exact same setup, just changing to selenium 4.0.0 and selenium 4.1.0 library and everything worked as expected. The issue is with selenium >4.1.1

**Comment by KazuCocoa** [comment]
Please use latest appium client that restricts the selenium version as 4.0 https://github.com/appium/python-client/pull/686
https://github.com/appium/python-client/issues/688 is an issue to update this client for selenium 4.1+

**Comment by KazuCocoa** [comment]
ah... your case was using latest appium client with selenium 4.1.1's WebDriverWait.  (not installed with appium cleint)

**Comment by Juanap12** [comment]
Yep! The issue appeared after installing selenium 4.1.1 or greater (also happening with selenium 4.1.3). Other problem would be that I can't downgrade the selenium version because 4.1.1 is the version that allows us to use web locators that aren't css selectors.

**Comment by KazuCocoa** [comment]
Could you try out https://pypi.org/project/Appium-Python-Client/2.2.0/ ?

**Comment by Juanap12** [comment]
Seems to be working now! Thank you very much for the fast response 👍
